dali/test/example_test.cc

```diff
@@ -20,38 +20,57 @@ namespace testing {
 
 
 class ExampleOperatorTestCase : public DaliOperatorTest {
+ protected:
+  ExampleOperatorTestCase() :
+          tlcpu(std::unique_ptr<TensorList<CPUBackend>>(new TensorList<CPUBackend>())),
+          tlgpu(std::unique_ptr<TensorList<GPUBackend>>(new TensorList<GPUBackend>())) {
+    std::call_once(once_flag_, [&]() {
```

Comment:
A constructor cannot have a race condition on data members. As such, std::call_once is unnecessary, as is the flag.